installation/standard.py

```diff
@@ -48,8 +48,8 @@
 #     If you set x11libdir (that is two ones and an el) AND x11include to point
 #     to the lib and include directories, they will be used.
 #     Otherwise a search is made for common locations.
-x11libdir=''
-x11include=''
+x11libdir='/usr/X11R6/lib'
+x11include='/usr/X11R6/include'
```

Comment:
I recompiled on crunchy and this is working????   The system libpng is found, so I don't know why buildbot did not like it.

```
configure:4516: checking for png_error in -lpng
configure:4541: /usr/bin/gcc -o conftest -I/export/nadeau1/uvcdat/09-09-15/Externals/include -I/export/nadeau1/uvcdat/09-09-15/Externals/lib/libffi-3.1/include   -fPIC -I/export/nadeau1/uvcdat/09-09-15/Externals/include -I/export/nadeau1/uvcdat/09-09-15/Externals/lib/libffi-3.1/include    -L/export/nadeau1/uvcdat/09-09-15/Externals/lib -L/export/nadeau1/uvcdat/09-09-15/Externals/lib64  -Wl,-rpath,/export/nadeau1/uvcdat/09-09-15/Externals/lib64 -Wl,-rpath,/export/nadeau1/uvcdat/09-09-15/lib -Wl,-rpath,/export/nadeau1/uvcdat/09-09-15/Externals/lib conftest.c -lpng -L/export/nadeau1/uvcdat/09-09-15/Externals/lib -L/usr/X11R6/lib  >&5
configure:4541: $? = 0
configure:4551: result: yes
```